endpoints/echo/Dockerfile.container-engine

```diff
@@ -0,0 +1,12 @@
+FROM debian:jessie
```

Comment:
Turns out we don't actually need this Dockerfile.  Removed, in a following commit.